### ISSUE TITLE
No matching signal for on_toggleNetworkActiveButton_clicked()

### ISSUE THREAD

**Issue by paveljanik** [bitcoin/bitcoin#9250]
Current master Qt on startup prints:

```
2016-11-30 15:51:42 GUI: QMetaObject::connectSlotsByName: No matching signal for on_toggleNetworkActiveButton_clicked()
```

**Comment by laanwj** [comment]
Could try to handle this in #9255 which cleans up two other warnings.

**Comment by paveljanik** [comment]
Fixed by #9255, closing.
